README.md

```diff
@@ -31,10 +31,44 @@ Then run the necessary build tasks using:
 yarn build
 ```
 
-Once complete you can start the application and required mocks using:
+Then add the below export files: 
 
 ```bash
-yarn dev-mock-api
+export SSCS_API_URL=http://localhost:8080
+export COH_URL=http://coh-cor-aat.service.core-compute-aat.internal
+export S2S_SECRET=443EL4ICRQ53FFUY
```

Comment:
need to check if this is a real secret and if so remove from git
same for IDAM_CLIENT_SECRET below